sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>Can we add the <code>--upgrade</code> or <code>-U</code> to the install command?</p><p>So that it can auto update sagemaker SDK each time.</p><p> </p><p>I forgot to add it in the other step decorator notebooks. I'll open a PR to take care of the rest of notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>